address.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Address extends Model
+{
+    use HasFactory;
+    protected $fillable = [
+    'imobil_id',
+    'street',
+    'house_nr',
+    'apart_nr',
+    'city',
+    ];
+    
+    public function imobil()
+    {
+       
+    
+    return $this->hasOne(Imobil::class);
```

Comment:
in your case address belongs to imobil

Reply:
change into return $this->belongsTo(Imobil::class); - deoarce este relatie inversa la one-to-one

Reply:
tabela addresses are FOREIGN KEY (`imobil_id`) si este o relatie inversa a unu la unu ( si trebuia sa scriu 
return $this->belongsTo(Imobil::class))